toolkits/landcover/ee_repo_sync.py

```diff
@@ -18,12 +18,12 @@
 import subprocess
 import os
 
-WORK_PATH = '~/.ee_repo_sync'
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+WORK_PATH = '%s/.gob' % SCRIPT_DIR
```

Comment:
Please add a comment as to what `gob` is.

Reply:
Renamed temp dir to a more intuitive name and added comment.